code/modules/reagents/chemistry/holder.dm

```diff
@@ -265,6 +265,7 @@
 			continue
 		if(!C)
 			C = R.holder.my_atom
+		sunset_process_reagent(C, R) //Sunset -- process species reagent
```

Comment:
This might cause double processing? It's been a long time but you might want to make sure